spec/factories.rb

```diff
@@ -20,6 +20,14 @@
     pod 1
     started_at_company Date.new(2017,01,01)
     person
+
+    trait :invalid do
```

Comment:
As mentioned above, traits are a great way to store complex changes to a factory and might not be the best use case here.